### PR TITLE
DM-23194: Expand out flags and Ra/Decl per band

### PR DESCRIPTION
**Pull request by yalsayyad**

### CODE REVIEW

File: yml/hsc.yaml [hunk 1]
```diff
@@ -551,48 +172,27 @@ tables:
...
-  - name: InputCount_flag
-    "@id": "#Object.InputCount_flag"
+  - name: calib_psf_used
+    "@id": "#Object.calib_psf_used"
     datatype: boolean
     description:
     mysql:datatype: BOOLEAN
-  - name: gBdChi2
-    "@id": "#Object.gBdChi2"
+  - name: gRa
```
Comment (hsinfang) [inline yml/hsc.yaml]: Having a set of Ra/Dec for each band is likely going to cause some problems in Qserv ingest/loading.  Is there one set that can be seen as the "canonical" coordinates for one object and it always exists? (Or maybe make an extra set?) Are the coordinates among the bands always close to each other for one object?

**Comment by hsinfang** [comment]
(deleted my previous comment. I was misled by some git diff....)

**Comment by hsinfang** [comment]
For one object, how different can its [griyz]Ra/Decl be from its coord_ra/coord_dec?  
It looks to me that we'll continue using coord_ra/coord_dec for Qserv ingest/loading, but does that mean we need to expect queries based on [griyz]Ra/Decl?

**Comment by yalsayyad** [comment]
Left comment on the Jira ticket, but spatial indexing should still only be on coord_ra and coord_dec. 

The DPDD specified per-band centroids: DCR? Off-center AGN? QA? Per-band centroids will always be within an arcsec.

**Pull request merged**